merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/SimulationService.java

```diff
@@ -3,13 +3,20 @@
 import java.util.List;
 import java.util.Map;
 
+import gov.nasa.jpl.aerie.merlin.driver.ActivityInstanceId;
+import gov.nasa.jpl.aerie.merlin.driver.SimulatedActivity;
 import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import gov.nasa.jpl.aerie.merlin.protocol.types.RealDynamics;
 import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue;
+import gov.nasa.jpl.aerie.merlin.protocol.types.ValueSchema;
 import gov.nasa.jpl.aerie.merlin.server.ResultsProtocol;
 import gov.nasa.jpl.aerie.merlin.server.models.PlanId;
 import org.apache.commons.lang3.tuple.Pair;
 
 public interface SimulationService {
   ResultsProtocol.State getSimulationResults(PlanId planId, RevisionData revisionData);
   Map<String, List<Pair<Duration, SerializedValue>>> getResourceSamples(PlanId planId, RevisionData revisionData);
+  Map<ActivityInstanceId, SimulatedActivity> getSimulatedActivities(PlanId planId, RevisionData revisionData);
+  Map<String, Pair<ValueSchema, List<Pair<Duration, SerializedValue>>>> getDiscreteProfiles(PlanId planId, RevisionData revisionData);
+  Map<String, List<Pair<Duration, RealDynamics>>> getRealProfiles(PlanId planId, RevisionData revisionData);
```

Comment:
Looks like you may need to rebase with `develop`. Maybe you have already and it's just not pushed :)

Reply:
Oops, sent too soon. Thanks!